docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -2,153 +2,150 @@
 
 If you need to import a new key into your wallet:
 
 ```shell
 dune --import-dev-key <PRIVATE_KEY>
 ```
 
-## Node Management
+## Node management
 
-Creating a new local EOS blockchain is easy with DUNE.
+DUNE easily creates a new local EOS blockchain.
+
+The following command creates a new node called `NODE_NAME`.  This node initiates with default settings. These default settings configure the node to serve as an API/producer node. You can deploy smart contracts to it and perform tests on it.
 
 ```shell
 dune --start <NODE_NAME>
 ```
 
-The command above creates a new node called `NODE_NAME` and starts it with default settings. 
-The node is configured to serve as an API/producer node that you can deploy smart contracts to, and perform tests on.
-
-> ❔ **Errors**
+> 📝 **In Case of Errors**
 >
 > You may see errors at the end of the node setup process.
-> If you do you can refer to this guide to troubleshoot common errors, or reach out to us on our
+> If you do, refer to this guide to troubleshoot common errors, or reach out to us on our
 > [Telegram channel](https://t.me/antelopedevs) for help.
 
-You can see a list of EOS nodes on your system:
+The following command shows you a list of EOS nodes on your system:
 
 ```shell
 dune --list
 ```
 
-You can also check if your active node's RPC API is live:
+The following command checks if your active node's RPC API is live:
 
 ```shell
 dune -- cleos get info
 ```
 
-To shut down your node:
+The following command shuts down your node:
 
 ```shell
 dune --stop <NODE_NAME>
 ```
 
-To remove a node entirely:
+The following command removes a node:
 
 ```shell
 dune --remove <NODE_NAME>
 ```
 
-
 ### Bootstrapping your environment
 
-There are a few system contracts that your development environment might need to rely on such as:
+There are a few system contracts that your development environment may need to rely on, such as:
 - `eosio.token` for **EOS** token transfers
 - `eosio.msig` for multisig transactions
 - `eosio.system` for system level actions such as resource management
 
-Bootstrapping your local node is easy, once you have an active node running, you can bootstrap it with:
+Bootstrapping your local node is easy. Once you have an active node running, bootstrap it with:
 
 ```shell
 dune --bootstrap-system-full
 ```
 
-
 ## Account management
 
-You deploy contracts on top of accounts, and also use them to interact with your smart contracts. 
+You deploy contracts on top of accounts. Use accounts to interact with your smart contracts. 
 
-To create a new account:
+The following command creates a new account:
 
 ```shell
 dune --create-account <ACCOUNT_NAME>
 ```
 
-To get account info:
+The following command gets account info:
 
 ```shell
 dune -- cleos get account <ACCOUNT_NAME>
 ```
+## Smart contract development
 
-## Smart Contract Development
+Create a sample project to learn how to use DUNE to compile, deploy, and interact with smart contracts.
```

Comment:
Our preference should always be first-person plural, or second-person point of view in guides. 

This is imperative point of view.

Reply:
This construction is second-person point of view. It is called "implied". "Implied subjects occur when a sentence does not state the doer of the action, but it is clear to whom the sentence is referring." This construction is a standard practice for instructions.